pledgeindia/pledge/urls.py

```diff
@@ -1,5 +1,7 @@
 from django.urls import path
-
+from .views import JsonData
 # urlpatterns
 
-urlpatterns = []
+urlpatterns = [
+    path("get/", JsonData, name="get-pledges"),
```

Comment:
This url should be more descriptive about what it does. `get` does not tell you anything.

Reply:
it is namespaced under 'pledge', so complete url will be '/pledge/get/', to get all pledges from database.

Reply:
Could you point me to the namespacing code? I cant seem to find it.

Reply:
It should be here, seems I forgot to add :/
Probably @omi10859 can include this
`path('pledge', include('pledge.urls', namespace='pledge')),`
in the main urlpatterns
https://github.com/tucosaurus/pledge-india/blob/master/pledgeindia/pledgeindia/urls.py#L20